src/catcher.ts

```diff
@@ -341,6 +343,19 @@ export default class Catcher {
     return this.release || null;
   }
 
+  /**
+   * Returns project id from integration token
+   */
+  private getProjectId(): string {
+    const decodedToken = jwtDecode<DecodedIntegrationToken>(this.token);
+
+    if (!decodedToken.projectId) {
+      throw new Error('Invalid JWT token. There is no project ID.');
```

Comment:
this Exception is not handled anywhere

Reply:
It will be displayed in dev tools console